### ISSUE TITLE
[ARM/Linux] Build failure - out-of-line definition of 'CreateDictionaryLookupHelper' does not match any declaration in 'DynamicHelpers'

### ISSUE THREAD

**Issue by parjong** [dotnet/coreclr#5910]
I encountered the following error while cross-building ARM CoreCLR:
```
/home/parjong/projects/toolchains/coreclr/src/vm/arm/stubs.cpp:3925:23: error: out-of-line definition of 'CreateDictionaryLookupHelper' does not match any declaration in 'DynamicHelpers'
PCODE DynamicHelpers::CreateDictionaryLookupHelper(LoaderAllocator * pAllocator, CORINFO_RUNTIME_LOOKUP * pLookup)
```

**Comment by parjong** [comment]
It seems that b9f5ae88cc49836c8d31f07db7800707165cdb06 changes the declaration of `CreateDictionaryLookupHelper`, but does not update its implementation for ARM (which is in src/vm/arm/stubs.cpp)

**Comment by adityamandaleeka** [comment]
Looks like this is being fixed with https://github.com/dotnet/coreclr/pull/5909

**Comment by parjong** [comment]
@adityamandaleeka It seems that #5909 also fixes this issue. I'll close this issue when #5909 is merged

**Comment by adityamandaleeka** [comment]
Fixed in #5909
